vms/example/xsvm/cmd/versionjson/cmd.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package versionjson
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/version"
+	"github.com/ava-labs/avalanchego/vms/example/xsvm"
+)
+
+type vmVersions struct {
+	Name       string            `json:"name"`
+	VMID       string            `json:"vmid"`
```

Comment:
Super small nit - but I think it probably makes sense not to hide the type behind the `string`ification. This shouldn't change the end result.
```suggestion
	VMID       ids.ID            `json:"vmid"`
```